client/state/plugins/installed/reducer.js

```diff
@@ -100,6 +101,8 @@ export const plugins = withSchemaValidation( pluginsSchema, ( state = {}, action
 		case PLUGINS_RECEIVE: {
 			return { ...state, [ action.siteId ]: action.data };
 		}
+		case PLUGINS_ALL_RECEIVE:
```

Comment:
This allows redux to be updated only once when the plugins are received in bulk instead of updating once per site.